python/pyarrow/types.pxi

```diff
@@ -15,6 +15,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import re
```

Comment:
This doesn't appear to be used anywhere.

Reply:
removed